src/vivarium_gates_mncnh/components/lbwsg.py

```diff
@@ -41,32 +38,18 @@
 GESTATIONAL_AGE = "gestational_age"
 
 
-class LBWSGDistribution(LBWSGDistribution_):
```

Comment:
This is no longer needed since the renaming of columns is handled in the artifact creaiton